deps/llvm.mk

```diff
@@ -487,6 +487,8 @@ $(eval $(call LLVM_PATCH,llvm-3.9.0_threads))
 $(eval $(call LLVM_PATCH,llvm-3.9.0_cygwin)) # R283427, Remove for 4.0
 $(eval $(call LLVM_PATCH,llvm-3.9.0_win64-reloc-dwarf))
 $(eval $(call LLVM_PATCH,llvm-3.9.0_D27296-libssp))
+$(eval $(call LLVM_PATCH,llvm-D27609-AArch64-UABS_G3))
```

Comment:
Yes. I have the patch copied over locally and I'm waiting for the other one to be reviewed.
The difference doesn't matter at all for us though.